assignments/arrays.js

```diff
@@ -75,30 +75,77 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+
+// function search(id, inventory){
+//   for (var i=0; i < inventory.length; i++) {
+//       if (inventory[33].id === 33) {
+//           return inventory[i];
+//       }
+//   }
+// }
+
+// console.log(inventory[32].car_year);
+
+// var resultObject = search("string 1", array);
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```

Comment:
what happens when items before this car get deleted?

definitely inventory[32] will no longer be the car with id 33. You can try a for loop and look for the id instead.

Reply:
Oh My mistake.
There is a for-loop in place in the search function. I just did not put it into good use. I will correct that.
Thanks!